core/src/text_serializer.cc

```diff
@@ -164,6 +171,14 @@ void SerializeFamily(std::ostream& out, const MetricFamily& family) {
         SerializeGauge(out, family, metric);
       }
       break;
+    // info is not handled by prometheus, we use gauge as workaround
+    // (https://github.com/OpenObservability/OpenMetrics/blob/98ae26c87b1c3bcf937909a880b32c8be643cc9b/specification/OpenMetrics.md#info-1)
+    case MetricType::Info:
+      out << "# TYPE " << family.name << " gauge\n";
```

Comment:
For reference, the [Java]((https://github.com/prometheus/client_java/blob/70e0f5ab5bf8f8876a18c75d33d1c9dc4cac9232/simpleclient_common/src/main/java/io/prometheus/client/exporter/common/TextFormat.java#L188-L189)) and [Python](https://github.com/prometheus/client_python/blob/c835d17749f53f19387aba09b419e4cc28eca320/prometheus_client/exposition.py#L204-L206) implementations. However, [Rust](https://github.com/prometheus/client_rust/blob/eef56cf5792694e6ca03c0193c59675a4d3622e1/src/encoding/text.rs#L749) seems to use info as type.